app/assets/javascripts/admin/addon/components/admin-notice.gjs

```diff
@@ -0,0 +1,29 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { htmlSafe } from "@ember/template";
+import ConditionalLoadingSection from "discourse/components/conditional-loading-section";
+import DButton from "discourse/components/d-button";
+import concatClass from "discourse/helpers/concat-class";
+import icon from "discourse-common/helpers/d-icon";
+import i18n from "discourse-common/helpers/i18n";
+
+export default class AdminNotice extends Component {
+  @action
+  dismiss() {
+    this.args.dismissCallback(this.args.problem);
+  }
+
+  <template>
+    <div class="notice">
+      <div class="message">
+        {{(if @icon (icon @icon))}}
+        {{htmlSafe @problem.message}}
+      </div>
+      <DButton
+        @action={{this.dismiss}}
+        @label="admin.dashboard.dismiss_notice"
+        class="btn-default"
```

Comment:
do we need btn-default?

Reply:
We did indeed not. I tried removing it and it's the same. 👍